autorel/clients/base.py

```diff
@@ -0,0 +1,109 @@
+import requests
+import json
+import sys
+
+__pyversion__ = sys.version_info[0]
+if __pyversion__ == 2: # Python2
+    from urlparse import urljoin
+else: # Python3
+    from urllib.parse import urljoin
+
+DEFAULT_HEADERS = {
+    'User-Agent' : 'syslog-ng-autorel',
+    'From' : 'ankprashar@gmail.com',
+    "Origin": 'black-perl.in'
+}
+
+DEFAULT_TIMEOUT = 30
+
+class BaseClient(object):
+    '''
+        A Base HTTP client
+    '''
+    http_methods = (
```

Comment:
Nope, it is not used anywhere. It is just a tuple to keep record of the methods supported by the http client.